### ISSUE TITLE
Gun in example scene has wrong hand transform direction

### ISSUE THREAD

**Issue by SmartCarrion** [TomorrowTodayLabs/NewtonVR#110]
On oculus, the gun points straight downward instead of forward.  (off by 90 degrees).  Using Jan 2nd build 76bf8b2

**Comment by keithbradner** [comment]
Yeah... we need to do some sort of separate pickup points for different controllers =/ I added it to trello
